packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -370,6 +370,31 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
       }
 
       this.context.updateClientState(clientState);
+
+      /* @conditional-compile-remove(call-transfer) */
+      const acceptedTransferCallState = this.context.getState().acceptedTransferCallState;
+
+      /* @conditional-compile-remove(call-transfer) */
+      // If there has been an accepted transfer call that is now in the connected state we should ensure we leave the current call.
+      // There is a possibility we accepted a transfer which the transferor thinks failed.
```

Comment:
This seems like it shouldn't be possible, does the SDK not throw an error/return a failure if the `call.accept()` is called after the timeout?

Reply:
No it does not. I will ask during office hours.